extratrees/src/main/resources/assets/extratrees/lang/en_US.lang

```diff
@@ -725,6 +725,112 @@ extratrees.fluid.miscfluid.sugarsyrup=Simple Syrup
 extratrees.fluid.miscfluid.agavenectar=Agave Nectar
 extratrees.fluid.miscfluid.grenadinesyrup=Grenadine Syrup
 
+## Fluid Tiles
```

Comment:
I would like to avoid duplicating these localization keys, can you think of a better way?

Reply:
That's how forestry does it. I don't think there's a way around it, it's due to how JEI shows fluid tiles.

Reply:
Hmm ok. We can fix this up some other time maybe, it's not a very big problem.